src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -213,6 +217,25 @@ protected final String getUriString(String filename) {
                 .toString();
     }
 
+    /**
+     * Performs verification of the file with the given file path using specified configuration
+     * and the array expected messages. Also performs verification of the config specified in
+     * input file.
+     *
+     * @param aConfig configuration.
+     * @param fileName file path to verify.
+     * @param expected an array of expected messages.
+     * @throws Exception if exception occurs during verification process.
+     */
+    protected final void verifyWithBddParser(Configuration aConfig,
```

Comment:
Please remove Bdd from everywhere in code.
We reuse concept but implementation of concept is inlined config in Input.

it should be `verifyWithInlinedConfig`.

Reply:
done